riscof/main.py

```diff
@@ -220,6 +229,18 @@ def execute():
         logger.info('Database File Generated: '+constants.framework_db)
         constants.env = args.env
         logger.info('Env path set to'+constants.env)
+    elif args.command=='run' :
+        if args.dbfile is None and args.testfile is None:
+            logger.info("Generating database for suite: "+args.suite)
+            work_dir = args.work_dir
+            constants.suite = args.suite
```

Comment:
The suite path is not being set correctly. Because of this when the framework is run from an intermediate start point to run the tests on the DUT, the default suite path is probed for the version number, whereas it should be the new suite path. This is because the `constants.suite` is used across modules to access the suite directory.

Reply:
Does it matter what the suite path is when running from an intermediate start point ?

Reply:
Yes. Because that suite path is used to report the version of the tests being used.

Reply:
I understood now. I'll update it.

Reply:
fixed.